helm/grafana/templates/dashboards-configmap.yaml

```diff
@@ -20,6 +20,6 @@ data:
       "basicAuth": false,
       "name": "prometheus",
       "type": "prometheus",
-      "url": "http://{{ printf "%s-%s" .Release.Name "prometheus" }}:9090"
+      "url": "http://prometheus-prometheus:9090"
```

Comment:
In my limited understanding, I understand, that multiple helm charts can be created as one release, therefore the templating from before was applicable.

Reply:
agreeed with @brancz.

Reply:
Ok.. But the datasource will need to be updated manually after installation because, Prometheus's service name is wrong...

Reply:
The idea is that it's not, when they're created through the same helm release.

Reply:
Ok. Got it... Will revert the commit